### PR TITLE
Fix quiz links

### PR DESCRIPTION
**Pull request by sklei2**
- Set the parents of the start quiz and quiz results to be the locker (so the back button at the top takes them there)

- Fixed a bug where the choice a user made for a reward was maintained for future reward choices.

### CODE REVIEW

File: app/src/main/java/swen_anigans/mathematicfanatic/QuizResultsActivity.java [hunk 1]
```diff
@@ -71,14 +71,8 @@ protected void onCreate(Bundle savedInstanceState) {
...
         Star[] starValues = getStars(correct,total,starGroup.getChildCount());
         setStars(starViews,starValues);
-        if(picked == null) {
-            setRewardListeners();
-        } else {
-            LinearLayout slideUp = (LinearLayout) findViewById(R.id.slideuppanel);
-            pickedTV = ((TextView) slideUp.getChildAt(0));
-            slideUp.removeAllViews();
-            formRewardSelectedLayout(slideUp,picked);
-        }
```
Comment (matthewGallagher) [inline app/src/main/java/swen_anigans/mathematicfanatic/QuizResultsActivity.java]: what was this doing before?

Comment (sklei2) [inline app/src/main/java/swen_anigans/mathematicfanatic/QuizResultsActivity.java]: @alecherbert

Comment (sklei2) [inline app/src/main/java/swen_anigans/mathematicfanatic/QuizResultsActivity.java]: I believe it was there to try and maintain state if the user selected something then slid the frame down. But I'm not entirely positive. I'm pretty positive it is working as intended now.

Comment (alecherbert) [inline app/src/main/java/swen_anigans/mathematicfanatic/QuizResultsActivity.java]: it was to make sure the reward stayed selected when the user changed activities & returned
- hit "go to locker" and returned"
- viewed video (playing the video counts as an activity change so when you go back from it, the oncreate gets called again)

Comment (sklei2) [inline app/src/main/java/swen_anigans/mathematicfanatic/QuizResultsActivity.java]: I do we want the reward to stay selected if the user leaves that activity? If the user goes to the locker they shouldn't be able to go back to that screen. They should have to either A) go through the quiz again to select a new reward or B) go to recess and view what they unlocked.

Comment (alecherbert) [inline app/src/main/java/swen_anigans/mathematicfanatic/QuizResultsActivity.java]: The returning from locker part was a side-effect of not having the parent-activity thing set up to force that back button functionality.

If I'm not mistaken, viewing the video directly at the quiz results, will cause the selected reward to be reset when the user stops viewing the video.  I can double check this later, but if it works without that chunk of code, I'll take your word for it.

Comment (sklei2) [inline app/src/main/java/swen_anigans/mathematicfanatic/QuizResultsActivity.java]: Ohhh! I didn't think of that case. Hold on let me see and then fix if it doesn't work as intended.

Comment (sklei2) [inline app/src/main/java/swen_anigans/mathematicfanatic/QuizResultsActivity.java]: Alright! Looked it over and it is now working!

**Pull request merged**